package.json

```diff
@@ -7,170 +7,116 @@
-    "@storybook/theming": "^7.1.0",
-    "@svgr/rollup": "^8.0.1",
-    "@svgr/webpack": "^8.0.1",
-    "@testing-library/dom": "^9.3.1",
-    "@testing-library/jest-dom": "^5.16.5",
+    "@storybook/theming": "^7.1.1",
+    "@testing-library/jest-dom": "^5.17.0",
     "@testing-library/react": "^14.0.0",
-    "@testing-library/react-hooks": "^8.0.1",
-    "@testing-library/user-event": "^14.4.3",
-    "@types/bluebird": "^3.5.38",
-    "@types/jest": "^27.4.1",
-    "@types/node": "^20.4.1",
-    "@types/react": "^18.2.14",
-    "@types/react-dom": "^18.2.6",
-    "@typescript-eslint/eslint-plugin": "^5.61.0",
-    "@typescript-eslint/parser": "^5.61.0",
-    "babel-core": "^6.26.3",
-    "babel-jest": "^29.3.1",
-    "babel-loader": "^8.2.5",
-    "babel-plugin-react-docgen": "^4.2.0",
-    "babel-plugin-react-require": "^3.1.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
-    "boxt": "^1.1.1",
-    "camelcase": "^7.0.1",
-    "chalk": "^4.1.2",
+    "@testing-library/react-hooks": "^8.0.0",
+    "@types/react-router": "^5.1.20",
+    "@types/react-router-dom": "^5.3.3",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
+    "@typescript-eslint/parser": "^6.2.0",
+    "autoprefixer": "^10.4.14",
+    "babel-loader": "^9.1.3",
+    "boxt": "^1.1.2",
+    "chalk": "^5.3.0",
     "css-loader": "^6.8.1",
-    "eslint": "^8.44.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-config-react-app": "^7.0.1",
-    "eslint-import-resolver-webpack": "^0.13.2",
-    "eslint-plugin-flowtype": "^8.0.3",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jest": "^27.2.2",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
-    "eslint-plugin-prettier": "^4.2.1",
-    "eslint-plugin-react": "^7.32.2",
+    "eslint": "^8.45.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-prettier": "^8.9.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.2.3",
+    "eslint-plugin-json": "^3.1.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-lodash": "^7.4.0",
+    "eslint-plugin-markdown": "^3.0.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-react": "^7.33.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "eslint-plugin-storybook": "^0.6.12",
-    "execa": "^5.1.1",
-    "file-loader": "^6.2.0",
-    "gh-pages": "^5.0.0",
+    "eslint-plugin-storybook": "^0.6.13",
+    "execa": "^7.2.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest": "^27.5.1",
-    "jest-environment-jsdom": "^27.5.1",
-    "jest-pnp-resolver": "^1.2.3",
-    "jest-watch-typeahead": "^1.1.0",
-    "lerna-changelog": "^2.2.0",
-    "monday-ui-style": "^0.1.172",
-    "postcss": "^8.4.25",
-    "postcss-scss": "^4.0.6",
-    "prettier": "^2.8.8",
-    "process": "^0.11.10",
+    "jest": "^29.6.2",
+    "jest-environment-jsdom": "^29.6.2",
+    "mini-css-extract-plugin": "^2.6.0",
+    "mocha": "^10.2.0",
+    "monday-ui-style": "^0.1.177",
```

Comment:
@SergeyRoyt - this is problematic because it creates a cycle.
`vibe-storybook-components` uses `monday-ui-style` to create the components, but `monday-ui-style` needs `vibe-storybook-components` to build documentation.

let's break the loop

Reply:
But it's dev dependency - it's needed only for the storybook, will it also bring the troubles? It shouldn't come with the distributed package, no?

Reply:
Cyclic dependencies are always dangerous and can cause unexpected behavior